travis-fix-coverage.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/bash
```

Comment:
@tyll, is this script necessary as it is not invoked within `tox.ini`?

Reply:
Please ignore this PR for now, it is not ready it I am in the middle of investigating it. A new version of python-coverage was released, that make allow a different approach.

Reply:
Ok, thank You. I walked through several commits and found that in the case of d945925 the results was reported successfully (https://coveralls.io/jobs/56225730).